monitors.yml

```diff
@@ -123,9 +123,6 @@ monitors:
     options:
       notify_no_data: true
       require_full_window: true
-      thresholds:
-        critical: 1.0
-        warning: 1.0
```

Comment:
I tried manually with a few values, including warning != critical, and still got client errors. You might want to do a few tests of your own to confirm.